src/core/terms.js

```diff
@@ -16,6 +16,19 @@
  * Copyright (c) 2018-2023 Open Assessment Technologies SA ;
  */
 
+/**
+ * Regex that matches the prefixed function operators
```

Comment:
Could you add an example of such a name to the comment? I'm not sure where to look.

Reply:
Sure, I will. It is related to a hack made in the `expr-eval` lib to use any functions with 2 params as binary parameters (the case if specific for the `nthrt` operator). The idea was to ease the composition of left-side exponent when rendering the expression. I think I'd do it differently today if I could have time for this, using intermediate parsing, but I did not want to refactor this part.